inc/class-command.php

```diff
@@ -52,6 +52,7 @@ protected function configure() {
     exec -- <command>             eg: exec -- vendor/bin/phpcs
 Open a shell:
     shell
+	ssh
```

Comment:
Just need to watch the white space here - use spaces rather than tabs for the docs

Reply:
Perfect, thanks! I will have to see if I can adjust my Atom settings to see these errors locally. Good to go now?